gulpfile.js

```diff
@@ -0,0 +1,872 @@
+                identifiers.length = 0;
+            }
+
+            for (i = 0; i < identifiers.length; ++i) {
+                if (identifiers[i].indexOf('//') >= 0 || identifiers[i].indexOf('/*') >= 0) {
+                    console.log(file + ' contains comments in the require list.  Skipping so nothing gets broken.');
+                    return;
+                }
+            }
+
+            var requires = [];
+
+            for (i = preserveFirst ? 1 : 0; i < names.length && i < identifiers.length; ++i) {
+                requires.push({
+                    name : names[i].trim(),
+                    identifier : identifiers[i].trim()
+                });
+            }
+
+            requires.sort(function(a, b) {
+                var aName = a.name.toLowerCase();
+                var bName = b.name.toLowerCase();
+                if (aName < bName) {
+                    return -1;
+                } else if (aName > bName) {
+                    return 1;
+                } else {
+                    return 0;
+                }
+            });
+
+            if (preserveFirst) {
+                requires.splice(0, 0, {
+                    name : names[0].trim(),
+                    identifier : identifiers[0].trim()
+                });
+            }
+
+            // Convert back to separate lists for the names and identifiers, and add
+            // any additional names or identifiers that don't have a corresponding pair.
+            var sortedNames = requires.map(function(item) {
+                return item.name;
+            });
+            for (i = sortedNames.length; i < names.length; ++i) {
+                sortedNames.push(names[i].trim());
+            }
+
+            var sortedIdentifiers = requires.map(function(item) {
+                return item.identifier;
+            });
+            for (i = sortedIdentifiers.length; i < identifiers.length; ++i) {
+                sortedIdentifiers.push(identifiers[i].trim());
+            }
+
+            var outputNames = ']';
+            if (sortedNames.length > 0) {
+                outputNames = '\r\n        ' +
+                              sortedNames.join(',\r\n        ') +
+                              '\r\n    ]';
+            }
+
+            var outputIdentifiers = '(';
+            if (sortedIdentifiers.length > 0) {
+                outputIdentifiers = '(\r\n        ' +
+                                    sortedIdentifiers.join(',\r\n        ');
+            }
+
+            contents = result[1] +
+                       outputNames +
+                       result[4].replace(/^[,\s]+/, ', ').trim() +
+                       outputIdentifiers +
+                       ') {' +
+                       result[6];
+
+            fs.writeFile(file, contents, callback);
+        });
+    }, done);
+});
+
+function combineCesium(debug, optimizer, combineOutput) {
+    return requirejsOptimize({
+        wrap : true,
```

Comment:
Should we bump the logLevel ?  As far as I can tell, the default from when we ran as a separate program was level 0 (TRACE) but now the default seems to be silent.

Up to you, but otherwise it would be nice to add some log statements saying what it's doing, because right now it sits silently for 10 minutes with no output at all.